spec/features/restaurant_spec.rb

```diff
@@ -166,13 +166,13 @@
     sign_in restaurant.user
     visit restaurant_path(restaurant)
 
-    expect(page).to have_content("Delete restaurant")
+    page.should have_selector(:link_or_button, 'Delete restaurant')
```

Comment:
this syntax is deprecated - should use `expect(page).to have_selector` instead of `page.should...`